task08/task08/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task08
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите х:");
+            Console.WriteLine("Результат: " + Function(double.Parse(Console.ReadLine())));
+            Console.ReadKey();
+        }
+        static double Function(double x)
+        {
+            double result = 0;
+            if (x > 2)
+            {
+                result = 2;
+            }
+            else if ((0 < x) & (x <= 2))
+            {
+                result = 0;
+            }
+            else if (x <= 0)
+            {
+                result = (-3) * x;
+            }
```

Comment:
Во второй клаузе сравнение с 2 лишнее. В третьей клаузе проверка уже не нужна. Также если тело клаузы состоит из одного оператора, скобки можно не писать.
```suggestion
            if (x > 2)
                result = 2;
            else if (0 < x)
                result = 0;
            else
                result = (-3) * x;
```